yt/utilities/tests/test_cosmology.py

```diff
@@ -216,7 +216,8 @@ def test_cosmology_calculator_answers():
     """
 
     fn = os.path.join(local_dir, "cosmology_answers.yml")
-    data = yaml.load(open(fn), Loader=yaml.FullLoader)
+    with open(fn) as fh:
+        data = yaml.load(fh, Loader=yaml.FullLoader)
```

Comment:
I missed that we were using `FullLoader`, hmmm.

Reply:
Is this a problem too ? do you want to make it a new issue ?

Reply:
nope